cli/src/command/run_dev.rs

```diff
@@ -50,21 +55,6 @@ pub struct RunDev {
 }
 
 impl RunDev {
-    fn artifact_path(&self, artifact_name: &str) -> PathBuf {
-        let mut path = self.artifacts_dir.clone();
-        path.push(artifact_name);
-        path
-    }
-
-    fn cleanup(&self) -> Result<(), Error> {
```

Comment:
I think `cleanup` should be called if `RunDev` is called with the corresponding flag (e.g., `--clean`). IMO, auto-clearing may make in some cases.

Reply:
Seems reasonable!